scripts/bb-dependencies.sh

```diff
@@ -45,7 +45,7 @@ Amazon*)
         zlib-devel libuuid-devel libblkid-devel libselinux-devel \
         xfsprogs-devel libattr-devel libacl-devel libudev-devel \
         device-mapper-devel openssl-devel elfutils-libelf-devel \
-        libffi-devel libaio-devel libmount-devel pam-devel \
+        libffi-devel libaio-devel libmount-devel pam-devel pamtester \
```

Comment:
`pamtester` doesn't appear to be packages for the Amazon AMIs.  I think it'd be OK to skip the testing on this platform, what I'd suggest is using the `log_unsupported()` function to skip these tests when `pamtester` isn't installed.

Reply:
Yes, `log_unsupported` is already used, here: https://github.com/openzfs/zfs/pull/9903/files#diff-3b90db4527561b0d1d7c4a77cb7ebc55R26
I think that this is correct.